diff -uNr monkey-0.6.3.orig/configure monkey-0.6.3/configure
--- monkey-0.6.3.orig/configure	Sun Apr  6 23:19:16 2003
+++ monkey-0.6.3/configure	Tue Apr  8 00:26:29 2003
@@ -298,7 +298,7 @@
 create_info()
 {
 cat > include/info.h <<EOF
-#define OS "$SYSNAME"
+#define OS "PLD/$SYSNAME"
 #define VERSION "$VERSION" /* Version de Monkey */
 #define MONKEY_PATH_CONF "$sysconfdir"
 EOF
diff -uNr monkey-0.6.3.orig/include/cgi.h monkey-0.6.3/include/cgi.h
--- monkey-0.6.3.orig/include/cgi.h	Mon Mar 24 15:56:05 2003
+++ monkey-0.6.3/include/cgi.h	Tue Apr  8 00:20:38 2003
@@ -25,4 +25,4 @@
 int 	change_dir(char *script);
 char	**env_set_basic(struct request *s_request);
 char	*env_add_var(char *name, const char *value);
-char	*cgi_alias(char *path, char *query, char *newstring );
\ No newline at end of file
+char	*cgi_alias(char *path, char *query, char *newstring );
diff -uNr monkey-0.6.3.orig/include/config.h monkey-0.6.3/include/config.h
--- monkey-0.6.3.orig/include/config.h	Sun Apr  6 18:00:30 2003
+++ monkey-0.6.3/include/config.h	Tue Apr  8 00:20:47 2003
@@ -60,4 +60,4 @@
 /* config.c */
 void	Monkey_Read_Config(char *path_conf);
 void	Monkey_Add_Index(char *indexname);
-void	config_error_msg(char *variable);
\ No newline at end of file
+void	config_error_msg(char *variable);
diff -uNr monkey-0.6.3.orig/include/deny.h monkey-0.6.3/include/deny.h
--- monkey-0.6.3.orig/include/deny.h	Mon Mar 24 15:56:05 2003
+++ monkey-0.6.3/include/deny.h	Tue Apr  8 00:20:53 2003
@@ -30,4 +30,4 @@
 
 void	Deny_Read_Config();
 void	Deny_Add(char *type, char *value);
-int     Deny_Check(char *uri); 
\ No newline at end of file
+int     Deny_Check(char *uri); 
diff -uNr monkey-0.6.3.orig/include/dir_html.h monkey-0.6.3/include/dir_html.h
--- monkey-0.6.3.orig/include/dir_html.h	Mon Mar 24 15:56:05 2003
+++ monkey-0.6.3/include/dir_html.h	Tue Apr  8 00:20:57 2003
@@ -19,4 +19,4 @@
 
 /* dir_html.c */
 int	 GetDir(char *dirname, int remote_fd, char *path_request, struct log_info *s_log, int protocol);
-char   *check_string(char *str);
\ No newline at end of file
+char   *check_string(char *str);
diff -uNr monkey-0.6.3.orig/include/monkey.h monkey-0.6.3/include/monkey.h
--- monkey-0.6.3.orig/include/monkey.h	Sun Apr  6 10:16:38 2003
+++ monkey-0.6.3/include/monkey.h	Tue Apr  8 00:21:20 2003
@@ -96,4 +96,4 @@
 #include "config.h"
 #include "chars.h"
 
-int mod_mysql_init();
\ No newline at end of file
+int mod_mysql_init();
diff -uNr monkey-0.6.3.orig/include/signals.h monkey-0.6.3/include/signals.h
--- monkey-0.6.3.orig/include/signals.h	Mon Mar 24 15:56:05 2003
+++ monkey-0.6.3/include/signals.h	Tue Apr  8 00:21:37 2003
@@ -19,4 +19,4 @@
 
 /* signals.c */
 void    signal_handler(int signo);
-void    Init_Signals();
\ No newline at end of file
+void    Init_Signals();
diff -uNr monkey-0.6.3.orig/include/user.h monkey-0.6.3/include/user.h
--- monkey-0.6.3.orig/include/user.h	Mon Mar 24 15:56:05 2003
+++ monkey-0.6.3/include/user.h	Tue Apr  8 00:21:45 2003
@@ -20,4 +20,4 @@
 /* user.c */
 int	User_main(struct request *sr); /* struct request */
 int	SetUIDGID();
-int	SetEGID_BACK();
\ No newline at end of file
+int	SetEGID_BACK();
diff -uNr monkey-0.6.3.orig/include/utils.h monkey-0.6.3/include/utils.h
--- monkey-0.6.3.orig/include/utils.h	Mon Apr  7 15:20:05 2003
+++ monkey-0.6.3/include/utils.h	Tue Apr  8 00:21:50 2003
@@ -35,4 +35,4 @@
 int	get_version_protocol(char *remote_protocol);
 char  *get_name_protocol(int remote_protocol);
 
-char *m_build_buffer(const char *format, ...);
\ No newline at end of file
+char *m_build_buffer(const char *format, ...);
diff -uNr monkey-0.6.3.orig/include/vhost.h monkey-0.6.3/include/vhost.h
--- monkey-0.6.3.orig/include/vhost.h	Sun Apr  6 18:00:30 2003
+++ monkey-0.6.3/include/vhost.h	Tue Apr  8 00:23:48 2003
@@ -34,6 +34,7 @@
 void	VHOST_Read_Config();
 void	VHOST_Config_Error();
 struct vhost	*VHOST_Find(char *host);
-int	 VHOST_Config_Add(char *vhost_servername, char *vhost_documentroot, 
-										char *vhost_cgi_alias, char *vhost_cgi_path, int forcegetdir);
-										
\ No newline at end of file
+int	 VHOST_Config_Add(char *vhost_servername, char *vhost_documentroot, char *vhost_cgi_alias, char *vhost_cgi_path, int forcegetdir);
+
+
+
