diff -burN monkey-0.7.1-orig/src/include/deny.h monkey-0.7.1/src/include/deny.h
--- monkey-0.7.1-orig/src/include/deny.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/deny.h	Mon Jun  9 12:20:27 2003
@@ -31,3 +31,4 @@
 void	Deny_Read_Config();
 void	Deny_Add(char *type, char *value);
 int     Deny_Check(char *uri); 
\ No newline at end of file
+
diff -burN monkey-0.7.1-orig/src/include/dir_html.h monkey-0.7.1/src/include/dir_html.h
--- monkey-0.7.1-orig/src/include/dir_html.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/dir_html.h	Mon Jun  9 12:20:10 2003
@@ -20,3 +20,4 @@
 /* dir_html.c */
 int	 GetDir(char *dirname, int remote_fd, char *path_request, struct log_info *s_log, int protocol);
 char   *check_string(char *str);
\ No newline at end of file
+
diff -burN monkey-0.7.1-orig/src/include/monkey.h monkey-0.7.1/src/include/monkey.h
--- monkey-0.7.1-orig/src/include/monkey.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/monkey.h	Mon Jun  9 12:22:21 2003
@@ -105,3 +105,4 @@
 void free_request(struct request *sr);
 
 #endif
\ No newline at end of file
+
diff -burN monkey-0.7.1-orig/src/include/request.h monkey-0.7.1/src/include/request.h
--- monkey-0.7.1-orig/src/include/request.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/request.h	Mon Jun  9 12:19:50 2003
@@ -123,3 +123,4 @@
 char	*Request_Find_Variable(char *request_body, char *string);
 void	Request_Error(int num_error, struct request *s_request, int debug, struct log_info *s_log);
 struct request	*Request_Strip_Header(struct request *sr, char *request_body);
+
diff -burN monkey-0.7.1-orig/src/include/signals.h monkey-0.7.1/src/include/signals.h
--- monkey-0.7.1-orig/src/include/signals.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/signals.h	Mon Jun  9 12:19:45 2003
@@ -20,3 +20,4 @@
 /* signals.c */
 void    signal_handler(int signo);
 void    Init_Signals();
\ No newline at end of file
+
diff -burN monkey-0.7.1-orig/src/include/support.h monkey-0.7.1/src/include/support.h
--- monkey-0.7.1-orig/src/include/support.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/support.h	Mon Jun  9 12:19:42 2003
@@ -24,3 +24,4 @@
 #define MONKEY_HTTP_PROTOCOL HTTP_11 /* Protocolo oficial */
 #define PROTOCOLS "HTTP/1.0 HTTP/1.1" /* Protocolos soportados */
 #define CGI_VERSION "1.1" /* Version del soporte CGI */
+
diff -burN monkey-0.7.1-orig/src/include/user.h monkey-0.7.1/src/include/user.h
--- monkey-0.7.1-orig/src/include/user.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/user.h	Mon Jun  9 12:19:39 2003
@@ -21,3 +21,4 @@
 int	User_main(struct request *sr); /* struct request */
 int	SetUIDGID();
 int	SetEGID_BACK();
\ No newline at end of file
+
diff -burN monkey-0.7.1-orig/src/include/utils.h monkey-0.7.1/src/include/utils.h
--- monkey-0.7.1-orig/src/include/utils.h	Sat May 31 05:40:40 2003
+++ monkey-0.7.1/src/include/utils.h	Mon Jun  9 12:20:23 2003
@@ -44,3 +44,4 @@
 void *M_malloc(size_t size);
 char *M_strdup(const char *s);
 void M_free(void *ptr);
\ No newline at end of file
+
--- monkey-0.7.0/src/include/vhost.h.orig	Mon May 12 18:17:48 2003
+++ monkey-0.7.0/src/include/vhost.h	Mon May 12 18:18:03 2003
@@ -37,4 +37,4 @@
 struct vhost	*VHOST_Find(char *host);
 int	 VHOST_Config_Add(char *vhost_servername, char *vhost_documentroot, 
 										char *vhost_cgi_alias, char *vhost_cgi_path, int forcegetdir);
-										
\ No newline at end of file
+
