--- monkey-0.7.0/src/include/cgi.h.orig	Mon May 12 18:13:50 2003
+++ monkey-0.7.0/src/include/cgi.h	Mon May 12 18:14:08 2003
@@ -46,3 +46,4 @@
 /* Childs managment */
 int	M_CGI_register_child(pthread_t thread, pid_t pid);
 int	M_CGI_free_childs(pthread_t thread, int exit_type);
+
--- monkey-0.7.0/src/include/deny.h.orig	Mon May 12 18:15:12 2003
+++ monkey-0.7.0/src/include/deny.h	Mon May 12 18:15:18 2003
@@ -30,4 +30,5 @@
 
 void	Deny_Read_Config();
 void	Deny_Add(char *type, char *value);
-int     Deny_Check(char *uri); 
\ No newline at end of file
+int     Deny_Check(char *uri); 
+
--- monkey-0.7.0/src/include/dir_html.h.orig	Mon May 12 18:15:34 2003
+++ monkey-0.7.0/src/include/dir_html.h	Mon May 12 18:15:39 2003
@@ -19,4 +19,5 @@
 
 /* dir_html.c */
 int	 GetDir(char *dirname, int remote_fd, char *path_request, struct log_info *s_log, int protocol);
-char   *check_string(char *str);
\ No newline at end of file
+char   *check_string(char *str);
+
--- monkey-0.7.0/src/include/monkey.h.orig	Mon May 12 18:16:19 2003
+++ monkey-0.7.0/src/include/monkey.h	Mon May 12 18:16:32 2003
@@ -94,4 +94,5 @@
 
 #include "modules.h"
 
-void free_request(struct request *sr);
\ No newline at end of file
+void free_request(struct request *sr);
+
--- monkey-0.7.0/src/include/signals.h.orig	Mon May 12 18:17:03 2003
+++ monkey-0.7.0/src/include/signals.h	Mon May 12 18:17:08 2003
@@ -19,4 +19,5 @@
 
 /* signals.c */
 void    signal_handler(int signo);
-void    Init_Signals();
\ No newline at end of file
+void    Init_Signals();
+
--- monkey-0.7.0/src/include/user.h.orig	Mon May 12 18:17:20 2003
+++ monkey-0.7.0/src/include/user.h	Mon May 12 18:17:26 2003
@@ -20,4 +20,5 @@
 /* user.c */
 int	User_main(struct request *sr); /* struct request */
 int	SetUIDGID();
-int	SetEGID_BACK();
\ No newline at end of file
+int	SetEGID_BACK();
+
--- monkey-0.7.0/src/include/utils.h.orig	Mon May 12 18:17:34 2003
+++ monkey-0.7.0/src/include/utils.h	Mon May 12 18:17:40 2003
@@ -43,4 +43,5 @@
 
 void *M_malloc(size_t size);
 char *M_strdup(const char *s);
-void M_free(void *ptr);
\ No newline at end of file
+void M_free(void *ptr);
+
--- monkey-0.7.0/src/include/vhost.h.orig	Mon May 12 18:17:48 2003
+++ monkey-0.7.0/src/include/vhost.h	Mon May 12 18:18:03 2003
@@ -37,4 +37,4 @@
 struct vhost	*VHOST_Find(char *host);
 int	 VHOST_Config_Add(char *vhost_servername, char *vhost_documentroot, 
 										char *vhost_cgi_alias, char *vhost_cgi_path, int forcegetdir);
-										
\ No newline at end of file
+
--- monkey-0.7.0/configure.orig	Mon May 12 18:07:36 2003
+++ monkey-0.7.0/configure	Mon May 12 18:07:50 2003
@@ -343,7 +343,7 @@
 create_info()
 {
 cat > $INCDIR/info.h <<EOF
-#define OS "$SYSNAME"
+#define OS "PLD/$SYSNAME"
 #define VERSION "$VERSION" /* Version de Monkey */
 #define MONKEY_PATH_CONF "$sysconfdir"
 EOF
